### PR TITLE
Support for solc 0.6.x

### PR DESCRIPTION
**Pull request by ggrieco-tob**

### CODE REVIEW

**Comment by incertia** [comment]
why must we disable the fallback test?

**Comment by ggrieco-tob** [comment]
Syntax for fallback functions is not compatible between 0.5.x and 0.6.x

File: .travis.yml [hunk 1]
```diff
@@ -50,8 +50,9 @@ script:
      stack --no-terminal test --ghc-options="$GHC_OPTIONS" $STACK_OPTS
    }
 
-   run_tests_using "solc-0.4.25"
+   #run_tests_using "solc-0.4.25"
```
Comment (ggrieco-tob) [inline .travis.yml]: This PR looks good to me. The only concern is the time spent testing with 0.4.x, 0.5.x and 0.5.x. Should we keep support for the oldest version?

Comment (ggrieco-tob) [reply .travis.yml]: Let's try to support it, if possible.

**Comment by incertia** [comment]
ill investigate what happened with multicontract

**Comment by ggrieco-tob** [comment]
@incertia I already merged with master, if you want to start digging into this.

**Comment by ggrieco-tob** [comment]
Superseeded by #571

**Pull request closed without merging**